bert/colossalai_utils/utils.py

```diff
@@ -22,40 +20,44 @@ def init_w_col(builder):
     print_log('Building data')
     train_data, test_data = build_data()
 
-    use_v2 = gpc.config.zero.pop('version', 2) == 2
-
-    cpu_offload = gpc.config.zero.offload_config.device == 'cpu'
+    use_zero = "zero" in gpc.config
+    if use_zero:
+        use_v2 = gpc.config.zero.pop('version', 2) == 2
+        cpu_offload = gpc.config.zero.offload_config.device == 'cpu'
+    else:
+        use_v2 = False
+        cpu_offload = None
 
     rank = get_rank()
     reset_peak_memory_stats(rank)
 
     print_log('Building model')
-    if use_v2:
+    if use_zero:
         shard_strategy = BucketTensorShardStrategy()
         with ZeroInitContext(convert_fp16='fp16' in gpc.config,

```

Comment:
The latest version colossal has removed the convert_fp16 arg.

Reply:
Done.